appstore-metadata-service/src/main/java/com/lgi/appstore/metadata/util/ApplicationTypeHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Liberty Global Technology Services BV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lgi.appstore.metadata.util;
+
+import com.lgi.appstore.metadata.model.ApplicationType;
+
+import java.util.Collection;
+import java.util.Locale;
+
+import static org.apache.commons.lang3.StringUtils.*;
+
+public final class ApplicationTypeHelper {
+    private ApplicationTypeHelper() {
+    }
+
+    public static ApplicationType fromValue(String type) {
+        if (isEmpty(type)) {
+            return null;
```

Comment:
ApplicationType is an enum, I don't think returning null is a good idea. I see null later on is mapped into native app. Maybe we can map it straightaway here?